feature/platform/tests/power_admin_down_up_test/power_admin_down_up_test.go

```diff
@@ -143,12 +146,13 @@ func powerDownUp(t *testing.T, dut *ondatra.DUTDevice, name string, cType oc.E_P
 	}
 	t.Logf("Component %s, power-admin-state after %f minutes: %v", name, time.Since(start).Minutes(), power)
 
-	oper, ok := gnmi.Await(t, dut, c.OperStatus().State(), timeout, oc.PlatformTypes_COMPONENT_OPER_STATUS_DISABLED).Val()
-	if !ok {
-		t.Errorf("Component %s oper-status, got: %v, want: %v", name, oper, oc.PlatformTypes_COMPONENT_OPER_STATUS_DISABLED)
+	if !deviations.ComponentPowerDownReturnsInactiveState(dut) {
```

Comment:
Can you add an explicit check for the inactive state based on this deviation flag (instead of skipping the oper status check altogether)?